packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -309,6 +310,19 @@ class CupertinoTextSelectionControls extends TextSelectionControls {
         );
     }
   }
+
+  @override
+  void handleCopy(TextSelectionDelegate delegate, ClipboardStatusNotifier? clipboardStatus) {
+    final TextEditingValue value = delegate.textEditingValue;
+    Clipboard.setData(ClipboardData(
+      text: value.selection.textInside(value.text),
+    ));
+    clipboardStatus?.update();
+    delegate.bringIntoView(delegate.textEditingValue.selection.extent);
+    // Hide the toolbar, but maintain the existing selection and keep the
+    // handles on the screen.
+    delegate.hideToolbar(true);
```

Comment:
I think ideally this should depend on what platform it is, since the material "copy" should have the same behavior on iOS.

Reply:
That's a good point.  I can get rid of this handleCopy override entirely then, and just put the switch statement in the main handleCopy for all platforms.